.github/workflows/ts-canary.yml

```diff
@@ -0,0 +1,24 @@
+name: TypeScript Canary
+
+on:
+  schedule:
+    - cron: '15 21 * * 4'
```

Comment:
Can you add a comment that describes the interval? Most people cannot read cron syntax.

Reply:
Done ✔️